multi-agent/run/scripts/migrate_builtin_system.py

```
@@ -0,0 +1,339 @@
+    user_configs_by_resource: dict = {}
+    for cfg_doc in user_configs_col.find({}):
+        resource_id = cfg_doc["resource_id"]
+        identity_key = cfg_doc["identity_key"]
+        fields = cfg_doc.get("fields", {})
+        try:
+            fields = _decrypt_fields(fields)
+        except Exception as exc:
+            raise RuntimeError(
+                f"Failed to decrypt overlay {resource_id}/{identity_key} — "
+                f"aborting before legacy data is modified: {exc}"
+            ) from exc
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)